docs/modules/servers/pages/distributed/architecture/consistency_model_data_replication_extend.adoc

```diff
@@ -0,0 +1,41 @@
+=== Cassandra consistency model
```

Comment:
The title here makes me confused. We have an another file called consistency-model.adoc. Why not putting these stuff in that file?

Reply:
I think what @vttranlina does here is that it allows him to inject what he wants. This is for cassandra in particular, I think there is a similar file for postgres.

Putting that in the consistency-model.adoc directly would be potentially hard to manage between Cassandra and Postgres, as those subsection would be quite different with both implementation.

@vttranlina am I correct?

Reply:
yes, absolute correct